dnf/cli/commands/repoquery.py

```diff
@@ -192,9 +192,9 @@ def set_argparser(parser):
                              help=_('show changelogs of the package'))
         outform.add_argument('--qf', "--queryformat", dest='queryformat',
                              default=QFORMAT_DEFAULT,
-                             help=_('format for showing found packages: '
-                                    '"%%{name} %%{version} ..". '
-                                    'Use --querytags to view full tag list'))
+                             help=_('display format for listing packages: '
+                                    '"%%{name} %%{version} ...", '
+                                    'use --querytags to view full tag list'))
```

Comment:
Could you please squash this with the commit it is amending and force-push for a clean set of commits? Thanks!

Reply:
Done,